app/assets/stylesheets/application.scss

```diff
@@ -35,6 +36,10 @@
   @import 'character_show_pages';
 
 //the default background image
+  #comments {
+    text-shadow: 2px 2px black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.